src/generateTeam.js

```diff
@@ -0,0 +1,141 @@
+const Manager = require("../lib/Manager");
+const Engineer = require("../lib/Engineer");
+const Intern = require("../lib/Intern");
+
+// function that generates card for team (Manager and/or Engineer and/or Intern)
+function generateTeamCards(team) {
+  let cards = [];
+  // loop through team members and generate relevant card for each
+  for (let i = 0; i < team.length; i++) {
```

Comment:
Consider replacing the for loop with .forEach instead e.g.
'''
team.forEach((employee) => { // logic goes in here })